src/app/router/Welcome/Welcome.tsx

```diff
@@ -85,7 +85,7 @@ function App() {
       )}
       <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
         <div className="text-center font-serif font-medium text-2xl pt-7 pb-3 dark:text-white">
-          <p>{t("heading")}</p>
+          <p>{t("heading") as string}</p>
```

Comment:
https://github.com/i18next/react-i18next/issues/1466#issuecomment-1092758879  
Might be possible to remove this cast once React-types or react-i18next is updated